math/CHANGELOG.md

```diff
@@ -11,28 +11,69 @@ Mention whether you follow Semantic Versioning.
 
 Usage:
 
-Change log entries are to be added to the Unreleased section under the
-appropriate stanza (see below). Each entry should ideally include a tag and
-the Github issue reference in the following format:
+Changelog entries are generated by git cliff ref: https://github.com/orhun/git-cliff
 
-* (<tag>) [#<issue-number>] Changelog message.
+Each commit should be conventional, the following message groups are supported.
 
-Types of changes (Stanzas):
+* feat: A new feature
+* fix: A bug fix
+* docs: Documentation only changes
+* style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+* refactor: A code change that neither fixes a bug nor adds a feature
+* perf: A code change that improves performance
+* test: Adding missing tests or correcting existing tests
+* build: Changes that affect the build system or external dependencies (example scopes: go, npm)
+* ci: Changes to our CI configuration files and scripts (example scopes: GH Actions)
+* chore: Other changes that don't modify src or test files
+* revert: Reverts a previous commit
 
-"Features" for new features.
-"Improvements" for changes in existing functionality.
-"Deprecated" for soon-to-be removed features.
-"Bug Fixes" for any bug fixes.
-"API Breaking" for breaking exported APIs used by developers building on SDK.
-Ref: https://keepachangelog.com/en/1.0.0/
+When a change is made that affects the API or state machine, the commit message prefix should be suffixed with `!`.
+
+Ref: https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json
 -->
 
 # Changelog
 
-## [Unreleased]
+## [math/v1.0.0-rc.0](https://github.com/cosmos/cosmos-sdk/releases/tag/math/v1.0.0-rc.0) - 2023-03-13
+
+### Features
+
+* Add rand funcs to math ([#15043](https://github.com/cosmos/cosmos-sdk/issues/15043))
```

Comment:
this is inverting the format, if we go with this we should update the docs.

Reply:
As this is a generated changelog, it is not supposed to be manually edited, so I think it's fine.
There are a few improvements to be made as discussed with @chatton, I've only added the git cliff to help us when tagging, to quickly verify whether the changelog is ok or not.

Reply:
its confusing that we have one way here and another elsewhere. Any chance we can decide on one or the other and go with it?

Reply:
This make sense, lmc how I can update the template!

Reply:
Updated